### PR TITLE
siro demo clean cg static graph oct 3 testing

### PR DESCRIPTION
**Pull request by jimmytyyang**
- A clean version of concept graph to improve the robustness of planning for the demo
- Result is documented here: https://docs.google.com/spreadsheets/d/1zINsVPK2SpKKEm10jtzacbgEY6bsDE3k6_UQ-9z_Y7Y/edit?gid=1598872581#gid=1598872581
- Contains fixes from Tushar for path planning
- Joint work with Tushar and Jimmy

### CODE REVIEW

**Review by tusharsangam — APPROVED** [review]
LGMT!

File: spot_rl_experiments/spot_rl/utils/utils.py [hunk 1]
```diff
@@ -178,7 +178,7 @@ def arr2str(arr):
 
 def calculate_height(object_tag):
     default_config = construct_config_for_semantic_place()
-    json_file_path = ROOT_PATH + "/sg_cache/cfslam_object_relations.json"
+    json_file_path = ROOT_PATH + "/sg_cache/cfslam_object_relations_cleaned.json"
```
Comment (jimmytyyang) [inline spot_rl_experiments/spot_rl/utils/utils.py:181]: remove this change? we should make the "cfslam_object_relations.json" as a clean version, and  then rename the old version as "cfslam_object_relations_old.json"

File: spot_rl_experiments/spot_rl/utils/stub_skills.py [hunk 2]
```diff
@@ -0,0 +1,102 @@
+import time
```
Comment (jimmytyyang) [inline spot_rl_experiments/spot_rl/utils/stub_skills.py:1]: I think we already have this file, could you please rebase it?

File: spot_rl_experiments/spot_rl/utils/skill_executor.py [hunk 3]
```diff
@@ -186,13 +186,13 @@ def execute_skills(self):
...
                 f"Navigation finished, succeded={succeded} , robot_holding={robot_holding}"
             )
-            if succeded and not robot_holding:
-                rospy.set_param("/is_arm_scanning", f"{str(time.time())},True")
-                time.sleep(1)
-                print("Scanning area with arm")
-                scan_arm(self.spotskillmanager.spot)
-            else:
-                print("Will not scan arm")
+            # if succeded and not robot_holding:
```
Comment (jimmytyyang) [inline spot_rl_experiments/spot_rl/utils/skill_executor.py:189]: remove this change

File: spot_rl_experiments/experiments/skill_test/test_robot_view_poses.py [hunk 4]
```diff
@@ -352,42 +353,280 @@
...
+        )
+    else:
+        spotskillmanager = None
 
-if in_fre_lab:
-    bboxs_info = bbox_info_static_graph
-else:
-    bboxs_info = bboxs_info_nyc
+    for receptacle_name in bboxs_info:
+        if receptacle_name in ["bed"]:
```
Comment (jimmytyyang) [inline spot_rl_experiments/experiments/skill_test/test_robot_view_poses.py:588]: remove this if statement

File: spot_rl_experiments/configs/cg_config.yaml [hunk 5]
```diff
@@ -1,9 +1,9 @@
-CG_ROOT_PATH: "data/2024-09-23--16-18-51-object_scan_fre_preprocessed"
+CG_ROOT_PATH: "/home/tushar/Desktop/spot-sim2real/data/2024-10-03--13-20-53-new_large_object_fre_preprocessed"
```
Comment (jimmytyyang) [inline spot_rl_experiments/configs/cg_config.yaml:1]: clean up this

File: spot_rl_experiments/configs/cg_config.yaml [hunk 6]
```diff
@@ -1,9 +1,9 @@
...
 # FRE: 2024-09-17--19-06-02_fre_apartment_shortscan_preprocessed / 2024-09-23--16-18-51-object_scan_fre_preprocessed
 # NYC: 2024-09-17--19-06-02_nyc_lab_preprocessed
 SCAN_TYPE: "r3d"
 USE_YOLO: False
-FILL_UP_GRID_LOCATION: [[86, 87, 94, 100], [75, 80, 99, 100], [2, 10, 18, 39], [26, 28, 4, 9], [85, 87,  30, 69]] # in array format: [y1:y2, x1:x2]. The value here is for Fremont apartment
+FILL_UP_GRID_LOCATION: [[0, 0, 0, 0]] # in array format: [y1:y2, x1:x2]. The value here is for Fremont apartment
```
Comment (jimmytyyang) [inline spot_rl_experiments/configs/cg_config.yaml:6]: clean up this?

File: spot_rl_experiments/spot_rl/utils/utils.py (hunk 1 above)
Comment (tusharsangam) [reply spot_rl_experiments/spot_rl/utils/utils.py:181]: sure makes sense to me

File: spot_rl_experiments/configs/cg_config.yaml [hunk 7]
```diff
@@ -1,11 +1,12 @@
-CG_ROOT_PATH: "data/2024-09-23--16-18-51-object_scan_fre_preprocessed"
+CG_ROOT_PATH: "data/2024-10-03--13-20-53-new_large_object_fre_preprocessed"
 # FRE: 2024-09-17--19-06-02_fre_apartment_shortscan_preprocessed / 2024-09-23--16-18-51-object_scan_fre_preprocessed
 # NYC: 2024-09-17--19-06-02_nyc_lab_preprocessed
+#FRE ROOT PATH : /home/tushar/Desktop/spot-sim2real/
```
Comment (jimmytyyang) [inline spot_rl_experiments/configs/cg_config.yaml:4]: remove your path

File: spot_rl_experiments/experiments/skill_test/test_robot_view_poses.py [hunk 8]
```diff
@@ -1,3 +1,4 @@
+# mypy: ignore-errors
```
Comment (jimmytyyang) [inline spot_rl_experiments/experiments/skill_test/test_robot_view_poses.py:1]: remove this

**Review by jimmytyyang — COMMENTED** [review]
looks good to me

**Pull request merged**